vispy/visuals/shaders/tests/test_function.py

```diff
@@ -241,6 +241,12 @@ def test_function_basics():
     assert_equal(fun.name, 'main')
     assert len(fun.template_vars) == 2
     assert_in('foo', fun.template_vars)
+
+    # Test that `var in fun` syntax works as well
+    assert 'foo' in fun
+    assert_in('foo', fun)
+    assert_in('bar', fun)
```

Comment:
I feel like with modern pytest the `assert_in` helper function isn't needed, right?

What is the difference between the first `in` test with foo and the `assert_in` version?

Could you also add a test for something that shouldn't be in it (a False condition).